android/engine/src/main/java/org/smartregister/fhircore/engine/data/local/register/dao/AncPatientRegisterDao.kt

```diff
@@ -30,12 +30,12 @@ import org.hl7.fhir.r4.model.Encounter
 import org.hl7.fhir.r4.model.Flag
```

Comment:
Will this class, and this entire package, be deleted by the final version of this PR?

Reply:
Yes. I will delete all the classes in this package and rename other classes as the last step of this code refactor.

Reply:
Noted.